frontend/pulse3d/src/pages/users-info.js

```diff
@@ -146,9 +146,13 @@ export default function UserInfo() {
   useEffect(() => {
     const newList = usersData.filter((user) => {
       //if the column containes date data
-      return formatDateTime(user[toUserField[filtercolumn]])
-        .toLocaleLowerCase()
-        .includes(filterString.toLocaleLowerCase());
+      if (["Date Created", "Last Loggedin"].includes(filtercolumn)) {
```

Comment:
you just removed "Date Created", "Last Loggedin" from the column names above? is that right?

Reply:
@luciipak  No that was in the Uploads table. This is in the User table

Reply:
This is on the users table, I think it's still there. We need to change `Loggedin` to something else since it is customer facing, I think we should make it `Last Login`

Reply:
Got it, my bad on that